mslice/models/cut/matplotlib_cut_plotter.py

```diff
@@ -57,13 +57,13 @@ def _create_cut(self, workspace):
         plt.gcf().canvas.draw()
 
     def set_icut(self, icut):
-        if hasattr(icut, 'plot_cut'):
+        if not icut:
```

Comment:
I think this would read a little better if it were:

```python
if icut is not None:
    if hasattr(icut, 'plot_cut'):
        ...
    else:
        ...
else:
    self.icut = None
```